tools/git-version.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.ajoberstar:grgit:1.9.2'
+    }
+}
+
+import org.ajoberstar.grgit.Grgit
+
+ext {
+    git = Grgit.open(currentDir: projectDir)
+    gitVersionName = ""
+    gitVersionName = git.describe()
```

Comment:
Ouch sorry, wrong copy&paste. `HEAD^` is to get the "previous" tag (it's what we use to obtain the changelog).

The actual command for the _current_ tag would be simpler: 
```
gitVersionName = 'git describe'.execute([], project.rootDir).text.trim()
```